### PR TITLE
Added cross-references to sort weight tables

### PR DESCRIPTION
**Pull request by rpetrusha**
## ## Added cross-references to sort weight tables

Fixes dotnet/docs#6731 

//cc @tarekgh @david-haemmerle

### CODE REVIEW

File: xml/System.Globalization/CompareOptions.xml [hunk 1]
```diff
@@ -45,12 +45,13 @@
...
+ The .NET Framework uses three distinct ways of sorting: word sort, string sort, and ordinal sort. Word sort performs a culture-sensitive comparison of strings. Certain nonalphanumeric characters might have special weights assigned to them. For example, the hyphen ("-") might have a very small weight assigned to it so that "coop" and "co-op" appear next to each other in a sorted list. String sort is similar to word sort, except that there are no special c0ases. Therefore, all nonalphanumeric symbols come before all alphanumeric characters. Ordinal sort compares strings based on the Unicode values of each element of the string.  For a downloadable set of text files that contain information on the character weights used in sorting and comparison operations for Windows operating systems, see [Sorting Weight Tables](https://www.microsoft.com/en-us/download/details.aspx?id=10921).
```
Comment (tarekgh) [inline xml/System.Globalization/CompareOptions.xml]: >c0ases [](start = 458, length = 6)

typo?

**Comment by tarekgh** [comment]
LGTM. 

CC @shawnsteele  if he has any comment here. 

@rpetrusha do we care to mention the same for ICU regarding .Net Core running on Linux and Mac OS? 

http://userguide.icu-project.org/collation
https://www.unicode.org/Public/UCA/latest/allkeys.txt

We can do that in other PR if you think it is worth to do it.

File: xml/System.Globalization/CompareInfo.xml [hunk 2]
```diff
@@ -76,7 +76,11 @@
  [!code-cpp[CompareInfo#1](~/samples/snippets/cpp/VS_Snippets_CLR/CompareInfo/cpp/CompareInfo.cpp#1)]
  [!code-csharp[CompareInfo#1](~/samples/snippets/csharp/VS_Snippets_CLR/CompareInfo/CS/CompareInfo.cs#1)]
  [!code-vb[CompareInfo#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/CompareInfo/VB/CompareInfo.vb#1)]  
-  
+
+## See also
+
+[Sorting Weight Tables for Windows operating systems](https://www.microsoft.com/en-us/download/details.aspx?id=10921)
```
[the remark this answers is not in the document]
Comment (rpetrusha) [reply xml/System.Globalization/CompareInfo.xml]: @tarekgh, It's definitely worth adding the ICU links. I'll create an issue to do it during the next sprint.

Thanks for the information on cross-references, @mairaw. I'll make the changes in this PR.

File: xml/System.Globalization/CompareOptions.xml [hunk 3]
```diff
@@ -45,12 +45,13 @@
...
+ The .NET Framework uses three distinct ways of sorting: word sort, string sort, and ordinal sort. Word sort performs a culture-sensitive comparison of strings. Certain nonalphanumeric characters might have special weights assigned to them. For example, the hyphen ("-") might have a very small weight assigned to it so that "coop" and "co-op" appear next to each other in a sorted list. String sort is similar to word sort, except that there are no special cases. Therefore, all nonalphanumeric symbols come before all alphanumeric characters. Ordinal sort compares strings based on the Unicode values of each element of the string.  For a downloadable set of text files that contain information on the character weights used in sorting and comparison operations for Windows operating systems, see [Sorting Weight Tables](https://www.microsoft.com/en-us/download/details.aspx?id=10921).
```
[the remark this answers is not in the document]
Comment (rpetrusha) [reply xml/System.Globalization/CompareOptions.xml]: .NET, definitely.

**Pull request merged**